core/designsystem/src/main/kotlin/org/mifospay/core/designsystem/component/PermissionBox.kt

```diff
@@ -115,7 +115,7 @@ fun PermissionBox(
 
     if (shouldShowPermissionRationale) {
         MifosDialogBox(
-            showDialogState = rememberSaveable { mutableStateOf(shouldShowPermissionRationale) },
+            showDialogState =  shouldShowPermissionRationale,
```

Comment:
There's a reason it should be rememberSaveable { mutableStateOf , it saves user preferance about permissions and doesn't start the whole process again